dpsim/src/PFSolver.cpp

```diff
@@ -321,6 +321,21 @@ void PFSolver::determineNodeBaseVoltages() {
             "Choose base voltage {}V of {} to convert pu-solution of {}.",
             baseVoltage_, gen->name(), node->name());
         break;
+      } else if (std::shared_ptr<CPS::SP::Ph1::Load> load =
+                     std::dynamic_pointer_cast<CPS::SP::Ph1::Load>(comp)) {
+        baseVoltage_ = load->attributeTyped<CPS::Real>("V_nom")->get();
+        SPDLOG_LOGGER_INFO(
+            mSLog, "Choose base voltage of {}V to convert pu-solution of {}.",
```

Comment:
I think we should add a space between numbers and their unit:
```suggestion
            mSLog, "Choose base voltage of {} V to convert pu-solution of {}.",
```

Reply:
Suggestion adopted